canonical/ubuntu.com
# Update copy on /support

## Done

- Updates based on [copydoc](https://docs.google.com/document/d/1AU2dwTXpQk2UuEvSuAqRKrbPWYb-LR0ZGvWu6xw95I8/edit#heading=h.pkvhhqabnqfc) changes and [design](https://www.figma.com/file/YdmmtZxcCdEVJE4JmSqfeD/Ubuntu.com%2Fsupport?type=design&node-id=233-3065&mode=design&t=teXUetDya3yo2y4d-0)

## QA

- Go to /support
- Check that the copy changes match
- Check the design matches

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10400

## Screenshots

![image](https://github.com/canonical/ubuntu.com/assets/58276363/3a7f960d-4d70-4ca9-aca6-7524beae2bfc)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
